tests/data/module_docstring_1.py

```diff
@@ -0,0 +1,25 @@
+"""Single line module-level docstring should be followed by single newline."""
```

Comment:
I don't think we'll run this test as it's not in `simple_cases/`.

Reply:
Moved the tests to ``preview/`` to make sure they run under preview mode. They executed now with `pytest -k module_docstring`.

Also added another two tests cases of which I thought the original PR didn't really have a good test:
1. A file that already has a new line in-between the docstring and the rest of code and
2. A file that has no lines in between the docstring and the rest of code and should thus get one